src/chi-vue/src/components/data-table/DataTableTooltip.tsx

```diff
@@ -22,41 +23,28 @@ export default class DataTableTooltip extends Vue {
 
   _isTruncated() {
     const wrapper = this.$refs.wrapper as HTMLElement;
-    const wrapperClone = wrapper.cloneNode(true) as HTMLElement;
-
-    wrapperClone.style.display = 'inline';
-    wrapperClone.style.width = 'auto';
-    wrapperClone.style.visibility = 'hidden';
-    wrapperClone.style.overflow = 'visible';
-
-    wrapper.parentNode?.appendChild(wrapperClone);
-
-    const isTruncated = wrapperClone.offsetWidth > wrapper.offsetWidth;
-
-    wrapperClone.parentNode?.removeChild(wrapperClone);
+    const isTruncated =
+      Math.abs(wrapper.scrollHeight - wrapper.clientHeight) > DATA_TABLE_CELL_THRESHOLD ||
+      wrapper.scrollWidth > wrapper.clientWidth;
```

Comment:
The ellipsis check for the width was something like this: e.offsetWidth < e.scrollWidth. Would it make sense to do the width check like this?